spec/src/main/asciidoc/rest-endpoints.adoc

```diff
@@ -254,8 +290,8 @@ If `GET /metrics/base` exposes multiple values like this:
 [source]
 ----
 {
-  "fooVal": 12345,
-  "barVal": 42
+  "fooVal{app=webshop}": 12345,
+  "barVal{app=webshop,component=backend}": 42
```

Comment:
need to include an example that has the same metric name more than once, with differing tag sets so that we can show how that would look below in the OPTIONS output